framework/src/transfers/MultiAppUserObjectTransfer.C

```diff
@@ -37,14 +39,20 @@ validParams<MultiAppUserObjectTransfer>()
       "user_object",
       "The UserObject you want to transfer values from.  Note: This might be a "
       "UserObject from your MultiApp's input file!");
-
+  params.addParam<bool>("injective_master_to_sub_map",
```

Comment:
I don't feel too strongly about this, but I had to go look up what 'injective' means. If there's another term that might be more widely understood, that could be helpful.  Maybe something like `all_master_nodes_contained_in_sub_app`?

Reply:
@permcody Any thoughts on the name for this parameter?

Reply:
I like the latter parameter name, but I'm ok with either.

Reply:
I have changed the name of this parameter to `all_master_nodes_contained_in_sub_app`. Also, it will now generate an error if master node is found in 2 or more sub apps and not just if the master node is not found in any sub app.